<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TextEditor</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>301</width>
     <height>198</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>888</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="Font">
    <property name="title">
     <string>Шрифт</string>
    </property>
    <addaction name="changeFontSize"/>
    <addaction name="changeFontStyle"/>
    <addaction name="separator"/>
    <addaction name="changeFontColor"/>
    <addaction name="changeBackgroundColor"/>
    <addaction name="changeBackgroundTextColor"/>
    <addaction name="separator"/>
    <addaction name="changeHighlight"/>
   </widget>
   <widget class="QMenu" name="File">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="createFile"/>
    <addaction name="openFile"/>
    <addaction name="separator"/>
    <addaction name="saveFile"/>
    <addaction name="saveNowFile"/>
   </widget>
   <widget class="QMenu" name="Compiler">
    <property name="title">
     <string>Компилятор</string>
    </property>
    <addaction name="run"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Подсветка синтаксиса</string>
    </property>
   </widget>
   <addaction name="File"/>
   <addaction name="Font"/>
   <addaction name="Compiler"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="createFile">
   <property name="text">
    <string>Создать новый</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="openFile">
   <property name="text">
    <string>Открыть файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveFile">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="changeFontSize">
   <property name="text">
    <string>Изменить размер текста</string>
   </property>
  </action>
  <action name="changeFontStyle">
   <property name="text">
    <string>Изменить шрифт</string>
   </property>
  </action>
  <action name="changeFontColor">
   <property name="text">
    <string>Изменить цвет текста</string>
   </property>
  </action>
  <action name="changeBackgroundColor">
   <property name="text">
    <string>Изменить цвет фона</string>
   </property>
  </action>
  <action name="changeBackgroundTextColor">
   <property name="text">
    <string>Изменить цвет фона текста</string>
   </property>
  </action>
  <action name="saveNowFile">
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="changeHighlight">
   <property name="text">
    <string>Изменить цвет выделения текста</string>
   </property>
  </action>
  <action name="run">
   <property name="text">
    <string>Собрать и запустить</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
